proj/scala-rewrites.conf

```diff
@@ -0,0 +1,36 @@
+// https://github.com/scala/scala-rewrites.git#master
+
+// This project is unusual in that we build it using only binary
+// dependencies, since the community build's scalafix version is too
+// old and we can't upgrade it because it has too many dependencies.
+// (And scala-rewrites is 2.12-only because it lives in
+// sbt-plugin-land.)
```

Comment:
sbt-plugin-land?  It's 2.12 only because Scalafix is 2.12 only, for some reason:

https://repo1.maven.org/maven2/ch/epfl/scala/scalafix_2.13/

Reply:
I thought Scalafix was 2.12 primarily because it typically lives in sbt-land, but perhaps I'm mistaken.

Reply:
You might be better informed than me.  I thought it was 2.12 for the longest time because Scalameta was 2.12 only, and now it's 2.12 only because no one's got round to upgrading it.